<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestsResults</class>
 <widget class="QWidget" name="TestsResults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tests results</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit, QPlainTextEdit, QComboBox, QRadioButton, QPushButton, QDateTimeEdit
{
font: 15px &quot;Times New Roman&quot;;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="ListOfTestsLabel">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>List of tests</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="testTableWidget">
         <property name="styleSheet">
          <string notr="true">QTableWidget
{
	font: 15px &quot;Times New Roman&quot;;
}

QHeaderView::section 
{
    background-color: silver;
    border: none;
	font-family: Times new Roman;
	font-size: 16px;
}

QTableWidget::item:selected
{ 
background-color: rgb(36, 36, 36);
color: white;
}
</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::DashLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="searchForCandidatesLabel">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Search for candidates</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="infoLabel3">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>by first name / last name / e-mail address / mark</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchCandidateInput"/>
       </item>
       <item>
        <widget class="QTableWidget" name="candidateResultTableWidget">
         <property name="styleSheet">
          <string notr="true">QTableWidget
{
	font: 15px &quot;Times New Roman&quot;;
}

QHeaderView::section 
{
    background-color: silver;
    border: none;
	font-family: Times new Roman;
	font-size: 16px;
}

QTableWidget::item:selected
{ 
background-color: rgb(36, 36, 36);
color: white;
}
</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::DashLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="infoLabel">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Click on a test to see all the candidates marks.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="infoLabel2">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Double click on a candidate to see his test answers.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
